### ISSUE TITLE
Staging Review finding: Page titles are identical

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#127615]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** BIO - Pingwind
**Product:** VA Form 21-8940 Veteran's Application for Increased Compensation Based on Unemployability
**Feature:** Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User can't identify an element or its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility), @erinrwhite (IA), @kkitagawa-bosch (content)

### Description

The page titles are the same throughout the entire flow:"Apply For Additional Compensation Based On Inability To Work | Veterans Affairs" . 

Page titles provide wayfinding information to users and can help identify when a new page has loaded, which is especially important for single page applications. 

## Recommended action

Update page titles to have affinity to the H1 on each page.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 2.4.2 Page Titled - Level A](https://www.w3.org/TR/WCAG22/#page-titled)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
